src/tutorials/0004-mutable-file-system/11.js

```diff
@@ -42,51 +50,51 @@ const validate = async (result, ipfs) => {
     return {
       fail: 'Your root directory isn\'t empty. Are you sure you ran the `files.rm` method on your `/some` directory with the option `{ recursive: true }`?',
       logDesc: 'Here are the current contents of your root directory:',
-      log: JSON.stringify(rootDirectoryContents, null, 2)
+      log: rootDirectoryContents.map(utils.format.ipfsObject)
```

Comment:
Can you please add this formatting option to the instructions for building tutorials?

Reply:
added